src/view/Pages/Patients/edit/EditPatient.js

```diff
@@ -58,25 +62,7 @@ const EditPatient = ({ patient, onCloseEditModal }) => {
                       onChange={handleInputChange}
                     />
                   </label>
-                  <label>Gender</label>
```

Comment:
Why gender is still showing as removed?

Reply:
for gender field will handle that on pms-38 which is create and edit modal  after we get pull of pms-47. once gender wil add then update request will work.